.github/actions/verify-token/src/validate.ts

```diff
@@ -24,11 +25,11 @@ export function validateGitHubFields(gho: githubObj): void {
     process.env.GITHUB_EVENT_NAME
   );
 
-  // event_path
+  // event_payload_sha256
   validateField(
-    "github.event_path",
-    gho.event_path,
-    process.env.GITHUB_EVENT_PATH
+    "github.event_payload_sha256",
+    gho.event_payload_sha256,
+    tscommon.safeFileSha256(process.env.GITHUB_EVENT_PAYLOAD || "")
```

Comment:
Hope we don't run into unexpected problems with this hardened code.  There is not a lot of interesting info in it https://github.com/laurentsimon/slsa-on-github-test/actions/runs/4601805495/jobs/8130089172 ifaict

Reply:
I tested this and the event payload doesn't change. It only includes info about what triggered the initial workflow
https://github.com/ianlewis/actions-test/actions/runs/4604876020